[2023-11-21T00:39:09.093+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-18T00:00:00+00:00 [queued]>
[2023-11-21T00:39:09.099+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-18T00:00:00+00:00 [queued]>
[2023-11-21T00:39:09.099+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-21T00:39:09.108+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-18 00:00:00+00:00
[2023-11-21T00:39:09.119+0000] {standard_task_runner.py:57} INFO - Started process 118 to run task
[2023-11-21T00:39:09.126+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'shelter_data_dag', 'load_data', 'scheduled__2023-11-18T00:00:00+00:00', '--job-id', '125', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp5w6005su']
[2023-11-21T00:39:09.138+0000] {standard_task_runner.py:85} INFO - Job 125: Subtask load_data
[2023-11-21T00:39:09.225+0000] {task_command.py:416} INFO - Running <TaskInstance: shelter_data_dag.load_data scheduled__2023-11-18T00:00:00+00:00 [running]> on host c6d84631e1b9
[2023-11-21T00:39:09.305+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='shelter_data_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-18T00:00:00+00:00'
[2023-11-21T00:39:12.724+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-21T00:39:12.755+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=shelter_data_dag, task_id=load_data, execution_date=20231118T000000, start_date=20231121T003909, end_date=20231121T003912
[2023-11-21T00:39:12.799+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-21T00:39:12.848+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-21T00:44:28.574+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-18T00:00:00+00:00 [queued]>
[2023-11-21T00:44:28.619+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-18T00:00:00+00:00 [queued]>
[2023-11-21T00:44:28.634+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-21T00:44:28.721+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-18 00:00:00+00:00
[2023-11-21T00:44:28.742+0000] {standard_task_runner.py:57} INFO - Started process 188 to run task
[2023-11-21T00:44:28.752+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'shelter_data_dag', 'load_data', 'scheduled__2023-11-18T00:00:00+00:00', '--job-id', '168', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp5ems5nfz']
[2023-11-21T00:44:28.758+0000] {standard_task_runner.py:85} INFO - Job 168: Subtask load_data
[2023-11-21T00:44:28.900+0000] {task_command.py:416} INFO - Running <TaskInstance: shelter_data_dag.load_data scheduled__2023-11-18T00:00:00+00:00 [running]> on host 51d09cf321f5
[2023-11-21T00:44:29.242+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='shelter_data_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-18T00:00:00+00:00'
[2023-11-21T00:44:32.801+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-21T00:44:32.847+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=shelter_data_dag, task_id=load_data, execution_date=20231118T000000, start_date=20231121T004428, end_date=20231121T004432
[2023-11-21T00:44:32.881+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-21T00:44:32.937+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-21T01:00:24.878+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-18T00:00:00+00:00 [queued]>
[2023-11-21T01:00:24.890+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: shelter_data_dag.load_data scheduled__2023-11-18T00:00:00+00:00 [queued]>
[2023-11-21T01:00:24.890+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-21T01:00:24.908+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data> on 2023-11-18 00:00:00+00:00
[2023-11-21T01:00:24.925+0000] {standard_task_runner.py:57} INFO - Started process 187 to run task
[2023-11-21T01:00:24.940+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'shelter_data_dag', 'load_data', 'scheduled__2023-11-18T00:00:00+00:00', '--job-id', '228', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpwstjuukx']
[2023-11-21T01:00:24.945+0000] {standard_task_runner.py:85} INFO - Job 228: Subtask load_data
[2023-11-21T01:00:25.040+0000] {task_command.py:416} INFO - Running <TaskInstance: shelter_data_dag.load_data scheduled__2023-11-18T00:00:00+00:00 [running]> on host e39dda9a9deb
[2023-11-21T01:00:25.147+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='shelter_data_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-11-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-18T00:00:00+00:00'
[2023-11-21T01:00:28.842+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-11-21T01:00:28.868+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=shelter_data_dag, task_id=load_data, execution_date=20231118T000000, start_date=20231121T010024, end_date=20231121T010028
[2023-11-21T01:00:28.947+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-21T01:00:29.029+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
